### PR TITLE
[#5829] improvement(CLI): Slightly misleading error message when group is not specified

### PR DESCRIPTION
**Pull request by Abyss-lord**
### What changes were proposed in this pull request?

Slightly missing leading error message when group is not specified, it should give some hints to user.

### Why are the changes needed?

Fix: #5829 

### Does this PR introduce _any_ user-facing change?

NO

### How was this patch tested?

```bash
bin/gcli.sh group create --metalake demo_metalake
# Missing --group option.

bin/gcli.sh group details --metalake demo_metalake
# Missing --group option.

bin/gcli.sh group delete --metalake demo_metalake
# Missing --group option.
```

### CODE REVIEW

**Comment by Abyss-lord** [comment]
hi @justinmclean @xunliu , could you plz review this pr when you have time?

File: clients/cli/src/main/java/org/apache/gravitino/cli/GravitinoCommandLine.java [hunk 1]
```diff
@@ -531,6 +531,16 @@ protected void handleGroupCommand() {
...
 
+    if (CommandActions.LIST.equals(command)) {
+      newListGroups(url, ignore, metalake).handle();
+      return;
+    }
+
+    if (Objects.isNull(group)) {
+      System.err.println("Group is undefined. Please use -g or --group to specify a group.");
+      return;
+    }
```
Comment (tengqm) [inline clients/cli/src/main/java/org/apache/gravitino/cli/GravitinoCommandLine.java:542]: See my comment to your PR #5921

File: clients/cli/src/main/java/org/apache/gravitino/cli/GravitinoCommandLine.java [hunk 2]
```diff
@@ -531,6 +531,16 @@ protected void handleGroupCommand() {
 
     Command.setAuthenticationMode(auth, userName);
 
+    if (CommandActions.LIST.equals(command)) {
+      newListGroups(url, ignore, metalake).handle();
+      return;
+    }
+
+    if (Objects.isNull(group)) {
```
Comment (justinmclean) [inline clients/cli/src/main/java/org/apache/gravitino/cli/GravitinoCommandLine.java:539]: Why not group == null?

File: clients/cli/src/main/java/org/apache/gravitino/cli/GravitinoCommandLine.java [hunk 3]
```diff
@@ -531,6 +531,16 @@ protected void handleGroupCommand() {
...
+      System.err.println("Group is undefined. Please use -g or --group to specify a group.");
```
Comment (justinmclean) [inline clients/cli/src/main/java/org/apache/gravitino/cli/GravitinoCommandLine.java:540]: I would just say "Missing --group option." as that is in line with other error messages.

Comment (Abyss-lord) [reply clients/cli/src/main/java/org/apache/gravitino/cli/GravitinoCommandLine.java]: fix, thank you, plz check again.

**Review by justinmclean — DISMISSED** [review]
Thanks again for your contribution.

**Review by justinmclean — APPROVED** [review]
Thanks again for your contributions.

**Pull request merged**